src/hera.cpp

```diff
@@ -394,18 +302,41 @@ evmc_result hera_execute(
       ensureCondition(_code.size() > 5, ContractValidationFailure, "Invalid contract or metering failed.");
     }
 
-    execute(context, _code, state_code, *msg, result, meterInterfaceGas);
+    /*
+     * Instantiate, execute, and extract results from the correct WASM engine
+     */
+    ExecutionResult vmresult;
```

Comment:
Why change this from `result`?

Reply:
naming-wise it's a bit more clear IMO.

Reply:
`ExecutionResult executionResult;` would be confusing i think.